backend/src/test/java/de/neuefische/backend/service/UserServiceTest.java

```diff
@@ -30,25 +30,25 @@ void addUser_expect_valid_user() {
         User givenUser = new User(null,"nick",new ArrayList<>());
 
         //when
-        when(userRepo.addUser(givenUser)).thenReturn(givenUser);
+        when(userRepo.save(givenUser)).thenReturn(givenUser);
         User result = userService.addUser(givenUser);
 
         //then
         assertEquals(givenUser,result);
-        verify(userRepo).addUser(givenUser);
+        verify(userRepo).save(givenUser);
     }
 
-    @Test
+    /*@Test
```

Comment:
Is that test going to be adapted aka fixed or is it to be removed entirely?
I couldn't tell!